loginsystem/newsfeed/CSS/style.css

```diff
@@ -0,0 +1,243 @@
+body {
```

Comment:
there should only be one white (enter tab) space after a class style. delete the extra white spaces between the different styles for different elements.